mypyc/test-data/irbuild-classes.test

```diff
@@ -2686,3 +2686,16 @@ L0:
     r6 = PyObject_VectorcallMethod(r3, r5, 9223372036854775812, 0)
     keep_alive r2, self, key, val
     return 1
+
+[case testInvalidMypycAttr]
+from mypy_extensions import mypyc_attr
+
+@mypyc_attr("allow_interpreted_subclasses", "invalid_arg")  # E: 'invalid_arg' is not a supported `mypyc_attr`
+class InvalidArg:
+    pass
+@mypyc_attr(invalid_kwarg=True)  # E: 'invalid_kwarg' is not a supported `mypyc_attr`
+class InvalidKwarg:
+    pass
+@mypyc_attr(str())  # E: All `mypyc_attr` args must be string literals.
```

Comment:
Do we actually want this? I noticed the current code ignores any values that arent string literals, and I thought an err would be better than a silent ignore, but should we just parse the non-literal string value like anything else in python would? 

I dunno, maybe I'm overthinking it and this is fine as-is